_posts/2015-08-18-How-katas-can-help-you-learn.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: post
+name: How-katas-can-help-you-learn
+title: How katas can help you learn
+date: 2015-08-18 09:35:00 +00:00
+author: Ana Nogal
+image:
+    src: /assets/img/custom/blog/baton_long.jpg
+    attribution:
+        text: Baton_long by Alain Delmas
+        href: https://commons.wikimedia.org/wiki/File:Baton_long.jpg
+tags:
+- Kata
+- TDD
+- iOS
+- Objective-C
+- Learning
+---
+In the last couple of weeks I have spent my time doing katas to improve my Objective-C. As you may remember, I even had an issue, [that you can revisit here](http://wp.me/p4i8Xl-7h).
+
+I've learn a lot from other katas too: for instance the RomanNumerals kata... You think it's a simple one, that you wan't learn mush from. But it's a good exercise  for your red -> green -> refactor cycle. It's a simple algorithm nothing new will come from here... Wasn't I wrong!!!!!
```

Comment:
But it's a good exercise for practicing the red - green - refactor cycle. 
The sentence 'It's a simple algorithm ... ' may be better placed before the sentence 'But it's a good exercise ...'